### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tpiros**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
+# Introduction
+This application showcases how to integrate the Cloudinary's React SDK. The sample project showcases the following functionality:
```
Comment (carlevison) [inline README.md:2]:
```suggestion
This project demonstrates how to use the [Cloudinary React SDK](https://cloudinary.com/documentation/react_integration) to create a photo album app. 

The sample project showcases the following functionality:
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+
+* Using the Upload Widget
```
Comment (carlevison) [inline README.md:4]:
```suggestion
* Using the [Upload Widget](https://cloudinary.com/documentation/upload_widget) to upload images to your product environment
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+* Uploading via API calls
```
Comment (carlevison) [inline README.md:5]:
```suggestion
* Using the [Upload API](https://cloudinary.com/documentation/image_upload_api_reference) to upload images to your product environment
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+* Displaying transformed images
```
Comment (carlevison) [inline README.md:6]:
```suggestion
* [Transforming](https://cloudinary.com/documentation/image_transformations) and [displaying](https://cloudinary.com/documentation/react_image_transformations#plugins) images
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
...
+
 # Setup
 
-Create an unsigned upload preset in your Cloudinary product environment.
+Create an unsigned upload preset in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.
```
Comment (carlevison) [inline README.md:10]:
```suggestion
[Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.
```

File: README.md [hunk 6]
```diff
@@ -11,4 +18,4 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
 
 # Run
 
-`npm run dev`
+From your terminal run `npm run dev` and follow the instructions to see this sample project.
```
Comment (carlevison) [inline README.md:21]:
```suggestion
From your terminal, in the **react-pa** folder, run: 

* `npm i`
* `npm run dev` 

Then, click the localhost link to open the app in your browser.

Try uploading images using each of the upload tabs, then see your images displayed in the Photo Album tab.

# Explore

* Take a look at the source code to understand how it works.  
* Discover more in the [Cloudinary Docs](https://cloudinary.com/documentation).
* Ask for help in our [Community Forum](https://community.cloudinary.com/), in [Discord](https://discord.gg/Cloudinary), or [raise a support request](https://support.cloudinary.com/hc/en-us/requests/new).

```

**Review by carlevison — COMMENTED** [review]
I've made some suggestions.  If you can accept those, I'll see how they look, and let you know if there's anything else to change.  Possibly add step numbers to the Setup steps..?

File: README.md [hunk 7]
```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
 
 # Run
 
-`npm run dev`
+From your terminal, in the **react-pa** folder, run:
```
Comment (tpiros) [inline README.md:23]: if they clone the repository the folder name will be react-photo-album so we should either state that or tell them to use the folder name they chose

File: README.md [hunk 8]
```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
 
 # Run
 
-`npm run dev`
+From your terminal, in the **react-pa** folder, run: 
+
+* `npm i`
+* `npm run dev` 
+
+Then, click the localhost link to open the app in your browser.
```
Comment (tpiros) [inline README.md:28]: "or open your browser and navigate to localhost:3000 or to the port displayed in the terminal"

File: README.md [hunk 9]
```diff
@@ -1,8 +1,17 @@
...
+The sample project showcases the following functionality:
+
+* Using the [Upload Widget](https://cloudinary.com/documentation/upload_widget) to upload images to your product environment
+* Using the [Upload API](https://cloudinary.com/documentation/image_upload_api_reference) to upload images to your product environment
+* [Transforming](https://cloudinary.com/documentation/image_transformations) and [displaying](https://cloudinary.com/documentation/react_image_transformations#plugins) images
+
 # Setup
 
-Create an unsigned upload preset in your Cloudinary product environment.
+[Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.
```
Comment (carlevison) [inline README.md:12]:
```suggestion
[Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make a note of the name of the unsigned upload preset as well as your Cloudinary cloud name.
```

File: README.md (hunk 7 above)
Comment (carlevison) [inline README.md:23]:
```suggestion
From your terminal, in the root folder of your project, run: 
```

File: README.md (hunk 8 above)
Comment (carlevison) [inline README.md:28]:
```suggestion
Then, click the localhost link to open the app in your browser, or open a browser and navigate to localhost:3000 or to the port displayed in the terminal.
```

File: README.md [hunk 10]
```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
...
+* `npm run dev` 
+
+Then, click the localhost link to open the app in your browser.
+
+Try uploading images using each of the upload tabs, then see your images displayed in the Photo Album tab.
+
+# Explore
+
+* Take a look at the source code to understand how it works.  
+* Discover more in the [Cloudinary Docs](https://cloudinary.com/documentation).
```
Comment (carlevison) [inline README.md:35]:
```suggestion
* Discover more features in the [Cloudinary Docs](https://cloudinary.com/documentation).
```

File: README.md [hunk 11]
```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
...
+* Ask for help in our [Community Forum](https://community.cloudinary.com/), in [Discord](https://discord.gg/Cloudinary), or [raise a support request](https://support.cloudinary.com/hc/en-us/requests/new).
+
```
Comment (carlevison) [inline README.md:37]:
```suggestion

Feel free to fork this repo as a starting point for your own React app, contribute to it, or star it if you like it!
```

File: README.md [hunk 12]
```diff
@@ -1,8 +1,17 @@
+# Introduction
+This project demonstrates how to use the [Cloudinary React SDK](https://cloudinary.com/documentation/react_integration) to create a photo album app. 
+
+The sample project showcases the following functionality:
+
+* Using the [Upload Widget](https://cloudinary.com/documentation/upload_widget) to upload images to your product environment
+* Using the [Upload API](https://cloudinary.com/documentation/image_upload_api_reference) to upload images to your product environment
```
Comment (carlevison) [inline README.md:7]:
```suggestion
* Using the [REST API](https://cloudinary.com/documentation/client_side_uploading#direct_call_to_the_api) to upload images to your product environment
```

**Review by carlevison — COMMENTED** [review]
Just spotted another change.

File: README.md (hunk 7 above)
Comment (carlevison) [reply README.md:23]: Addressed in a suggestion further down.

File: README.md (hunk 8 above)
Comment (carlevison) [reply README.md:28]: Addressed in a suggestion further down.

**Pull request merged**